guides/main_concepts/indexes.md

```diff
@@ -97,6 +96,7 @@ By default, all fields attribute are added to the displayed-attributes list. If
 
 This list can be restricted to a certain set of attributes that you chose in the settings. That way, you can determine which fields should be not be returned upon search.
 
-### Stored attributes
+
```

Comment:
Why this blank space?

Reply:
After this little thing, I think it's ok

Reply:
```suggestion
```

Reply:
Yes sorry, I should have done it.